src/test/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorServiceSpec.scala

```diff
@@ -217,6 +221,56 @@ class PolicyEvaluatorServiceSpec extends FlatSpec with Matchers with TestSupport
     res.unsafeRunSync()
   }
 
+  "hasPermissionByUserEmail" should "return false if given action is not allowed for a user" in {
```

Comment:
Is there any value to having a positive test here?

Reply:
Right! Thanks for the suggestion!